src/main/grammars/RustLexer.flex

```diff
@@ -267,9 +265,18 @@ EOL_DOC_LINE  = {LINE_WS}*!(!("///".*)|("////".*))
                                     zzPostponedMarkedPos = zzStartRead;
                                     zzShaStride          = yylength() - 2; }
 
+//nessesary to allow proper restart of lexer when there is an error in raw literal, see RsRestartLexingTestCase
```

Comment:
Hmm, it looks like the tests from the `RsRestartLexingTestCase` pass even if you comment out this code.

Reply:
Weird... for me second test does successfully fail. Are you sure you updated generated lexer? Because running tests from gutter icon does not automatically update it.

Reply:
Yes, I tried `Generate parser` configuration and `:verification:test` gradle task.
@vlad20012 can you check these tests?

Reply:
For me it works as @rrevenantt expects: `test lexer restart in invalid raw literal` fails if comment these lines